uclibc defines __GLIBC__ but it does not expose struct shed_param as much as glibc
and is not needed too per standard. gnulib attempts to use it but we have to account
for it because in this case uclibc does not behave like glibc.

-Khem

Index: gettext-0.18/gettext-tools/gnulib-lib/spawn.in.h
===================================================================
--- gettext-0.18.orig/gettext-tools/gnulib-lib/spawn.in.h	2010-06-01 17:20:09.036427575 -0700
+++ gettext-0.18/gettext-tools/gnulib-lib/spawn.in.h	2010-06-01 17:20:38.856613943 -0700
@@ -31,7 +31,7 @@
 
 /* Get definitions of 'struct sched_param' and 'sigset_t'.
    But avoid namespace pollution on glibc systems.  */
-#ifndef __GLIBC__
+#if !defined __GLIBC__ || defined __UCLIBC__
 # include <sched.h>
 # include <signal.h>
 #endif
